domain-models-runtime/src/main/java/org/folio/rest/RestVerticle.java

```diff
@@ -251,12 +251,13 @@ public void start(Promise<Void> startPromise) throws Exception {
     final BodyHandler handler = BodyHandler.create();
 
     // IMPORTANT!!!
-    // the body of the request will be read into memory for ALL PUT requests
+    // the body of the request will be read into memory for ALL PUT, ALL PATCH requests
```

Comment:
Okapi does not support PATCH yet: https://github.com/folio-org/okapi/blob/60b32aa929dc0d2726671e814ba95a0293b9bd13/okapi-core/src/main/java/org/folio/okapi/util/CorsHelper.java#L47
Please create a Jira for RMB and a Jira for Okapi.

Reply:
Thank you for creating https://issues.folio.org/browse/RMB-636 .
Do you want to create the Okapi Jira yourself, or do you want us to do it?